dds_web/api/project.py

```diff
@@ -334,9 +336,13 @@ def get(self):
                 "usage": total_bhours_db,
                 "cost": total_cost_db,
             },
-            "total_size": total_size,
         }
 
+        if (
+            current_user.role == "Researcher" and p.current_status == "Available"
+        ) or current_user.role != "Researcher":
+            return_info["total_size"] = total_size
+
```

Comment:
Suggestion: You should just be able to do:
```
if total_size:
    return_info["total_size"] = total_size
```
because if the user is a researcher the total_size should be 0.

Reply:
I have switched the condition in the "if" statement to "total_size".

Reply:
Great, thanks!